### PR TITLE
Release fixes

### LINKED ISSUE #3058: Snags - round 2
Snags - round 2 

High Priority:

"Ubuntu Server for IBM Z and LinuxONE" becomes /download/server/s390x: https://docs.google.com/document/d/1d85IfqqxZYYZ0orj5lBhqLWLGpLYWRLdvs75lYsplGs/edit

Add redirect for /telco on the /telecommunications page. Resolve note in copydoc once complete: https://docs.google.com/document/d/15FY_IaT39V81FQcoSoEJpmRRjBX80PJ7uxjgzHnDDws/edit

Medium Priority:

Add a re-direct for /desktop/1710 to /desktop/ Please resolve comment in copydoc when complete: https://docs.google.com/document/d/1ZhMkLlkXcimHTIe1taXy24vbbR9pBkZd49H2sJqeQH0/edit 

Low Priority:

Link change on /cloud/thank-you: https://docs.google.com/document/d/1-dWhd0ydDWlYg16rEybPPmkQ6cdxsBmiFOuUz0ApCBM/edit

One small text edit on /kubernetes page: https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Final changes

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Ensure matches changes requested in below issue

## Issue / Card

Fixes #3058

### CODE REVIEW

**Comment by frankiegranato** [comment]
"Ubuntu Server for IBM Z and LinuxONE" becomes "s390x" in menu please

File: templates/shared/_thank_you_footer.html [hunk 1]
```diff
@@ -8,7 +8,7 @@ <h2>More about our enterprise services</h2>
     <div class="col-4 p-divider__block">
       <h3 class="p-heading--four">Ubuntu Advantage</h3>
       <p>Purchase our desktop support and access Ubuntu experts whenever you need.</p>
-      <p><a href="/support/plans-and-pricing">Learn about Ubuntu Advantage&nbsp;&rsaquo;</a></p>
+      <p><a href="https://buy.ubuntu.com">Buy Ubuntu Advantage&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/shared/_thank_you_footer.html]: This should be external

**Comment by frankiegranato** [comment]
All checked with Ant. +1

**Comment by steverydz** [comment]
@anthonydillon Fixed
@frankiegranato I've fixed the redirects. [See comment](https://github.com/canonical-websites/www.ubuntu.com/issues/3058#issuecomment-384408774)

**Pull request merged**